features/step_definitions/wallets-settings-utxos-steps.js

```diff
@@ -0,0 +1,72 @@
+import { Then } from 'cucumber';
+import { expect } from 'chai';
+import { BigNumber } from 'bignumber.js';
+import { getVisibleTextsForSelector } from '../support/helpers/shared-helpers';
+import { getWalletUtxosTotalAmount } from '../../source/renderer/app/utils/utxoUtils';
+import { formattedWalletAmount } from '../../source/renderer/app/utils/formatters';
+
+const component = '.WalletUtxoSettings_component';
+
+const selectors = {
+  component,
+  title: `${component} > h1`,
+  description: `${component} > p`,
+  chart: '.WalletUtxoSettings_responsiveContainer',
+  walletAmount: `${component} > p b:nth-child(1)`,
+  walletUtxosAmount: `${component} > p b:nth-child(2)`,
+};
+
+Then('the {string} element renders the following text:', async function(
+  element,
+  data
+) {
+  const [expectedTextData] = data.hashes();
+  const [renderedText] = await getVisibleTextsForSelector(
+    this.client,
+    selectors[element]
+  );
+  const expectedText = await this.intl(expectedTextData.message);
+  expect(renderedText).to.equal(expectedText);
+});
+
+Then(
+  'the page description displays the correct wallet and UTXOs amount',
+  async function() {
+    const [renderedWalletAmount] = await getVisibleTextsForSelector(
+      this.client,
+      selectors.walletAmount
+    );
+    const [renderedWalletUtxosAmount] = await getVisibleTextsForSelector(
+      this.client,
+      selectors.walletUtxosAmount
+    );
+
+    // TODO: Get the real active wallet amount
```

Comment:
@daniloprates what about this TODO?

Reply:
I'm having issues with this part. For some reason, in the test, `daedalus.stores.wallets.active` will return a BigNumber without its regular methods, but with Objects instead:

```shell
walletLovelaceAmount.toFormat: {}
typeof walletLovelaceAmount.toFormat: object
```

![image](https://user-images.githubusercontent.com/1504716/55733303-94ca8e00-59eb-11e9-8614-04490d2f3def.png)

Reply:
Interesting... We might need @DominikGuzei's help on this one 🙏

Reply:
Hm … no idea why that happens, would have to debug this

Reply:
@daniloprates did you figure this one out?

Reply:
@daniloprates ping!